### PR TITLE
adding windows build

### PR DESCRIPTION
**Pull request by DerThorsten**
A first very naive attempt to add a Windows build

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by DerThorsten** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D DBUILD_SHARED_LIBS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
```
Comment (jjerphan) [inline recipe/bld.bat:1]: Boost 1.70 introduces "native" CMake config scripts which might be accidentally used.

```suggestion
cmake -G "NMake Makefiles" ^
      -DBoost_NO_BOOST_CMAKE=ON ^
      -D DBUILD_SHARED_LIBS=OFF ^
      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
      %SRC_DIR%
```

**Comment by xhochy** [comment]
@DerThorsten @jjerphan Would you mind rebasing this? I got the feedstock uptodate again, so it would be nice to see which error appears now.

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by jjerphan** [comment]
Thank you, @xhochy.

**Pull request merged**